### ISSUE TITLE
apt-get update not finding proper repositories

### ISSUE THREAD

**Issue by timmeade** [davehorton/docker-drachtio-server#2]
While using the Dockerfile i am getting:

```
W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease  Unable to find expected entry 'main/binary-amd64/Packages' in Release file (Wrong sources.list entry or malformed file)

E: Some index files failed to download. They have been ignored, or old ones used instead.
```

easy fix:

add 

RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list

before the apt-get update line.

**Comment by davehorton** [comment]
actually, I probably need to deprecate this repo, because the preferred Dockerfile is in the drachtio-server project itself.  Please try that one

**Comment by timmeade** [comment]
OK Did not know that.  Just got it go through.  Had to add a build for cmake.  Was posting it as we speak.

**Comment by davehorton** [comment]
yes go ahead and make a PR if you like.  There may be a reason to keep this repo around, not sure though

**Comment by timmeade** [comment]
I will have to finish that compile.   I switched gears to the one in the server repo.

It built the new branch and it appears the CDRs are working better.   I see multiple attempts, starts, and stops    I’ll do some more testing in a bit.

Thanks

Tim.

**Comment by davehorton** [comment]
ok, on the cdrs for a b2b you should see attempt, start, and stop records for the A and B legs separately. Please update that other ticket with your results once you have them.  Much appreciated!
